plugins/pom-analyzer/src/main/java/eu/f4sten/pomanalyzer/CleanUpM2RepositoryArgs.java

```diff
@@ -0,0 +1,10 @@
+package eu.f4sten.pomanalyzer;
+
+import com.beust.jcommander.Parameter;
+
+public class CleanUpM2RepositoryArgs {
+
+    @Parameter(names= "--m2repo.path", arity = 1)
+    public String m2RepositoryPath = "~/.m2/repository";
```

Comment:
Please rename field to `pathM2` (to stay consistent with the naming of the argument.)